### DISCUSSION TITLE
How can I create a computed column with two columns of a splited table (by column)?

### DISCUSSION THREAD

**Discussion by DanielaSanabriaG** [fbaligand/kibana-enhanced-table#225] [Q&A]
Hello, 

I have a table with columns divided as a date histogram.

![image](https://user-images.githubusercontent.com/66104995/130371822-39f2a240-4db4-4183-bb70-93a900641d91.png)

and I want to compute the change in the column 'Rejected txs' between my two divisions. 

![image](https://user-images.githubusercontent.com/66104995/130371856-710800b4-2d8a-4e32-a4b7-836a1a5d7dd0.png)

Can I do this?

Thanks in advance

**Comment by fbaligand** [comment]
You cannot make calculations between several tables. 
In a computed column formula, you can only reference current table columns.
Sorry
